gettsim/pre_processing/piecewise_functions.py

```diff
@@ -24,22 +28,48 @@ def piecewise_linear(
 
```

Comment:
The docstring mentions `linear function`, while the func itself is now named `piecewise polynominal`.

Reply:
Changed docstring